apps/nextjs-website/react-components/types/PreFooter/PreFooter.ts

```diff
@@ -0,0 +1,20 @@
+import { CommonProps, Generic } from '../common/Common.types';
+
+type ImgProps = React.DetailedHTMLProps<
+  React.ImgHTMLAttributes<HTMLImageElement>,
+  HTMLImageElement
+>;
+
+export interface StoreButtonsProps {
+  readonly hrefGoogle?: string;
+  readonly hrefApple?: string;
+}
+
+export interface PreFooterProps extends CommonProps, PreFooterContentProps {
+  readonly storeButtons?: StoreButtonsProps;
+  decoration?: ImgProps | Generic | { url: string };
```

Comment:
I'd make the typing stricter, also due to the comment in the `PreFooter.tsx` file.